.github/workflows/build-test-measure.yml

```diff
@@ -577,6 +577,7 @@ jobs:
     needs: pre-run
     runs-on: ubuntu-latest
     if: needs.pre-run.outputs.changed-php-count > 0
+    continue-on-error: true
```

Comment:
Why specifically?

Reply:
Sometimes external-http tests may fail the workflow, causing main workflow to exit early.